jpw1991/chebs-mercenaries
# 2.2.3: Permit PvP - if enabled, minions will attack players and creat…

…ures that are not their owner.

Related PR in [Cheb's Necromancy](https://github.com/jpw1991/chebs-necromancy/pull/250)

To test in single player:

- Enable PvP in config (by default, it's off)
- Create a character eg. Modtest
- Log into your test world
- Make some minions, claim them so they're yours
- Log out, make new character, join world
- See if minions that belong to your previous character attack you
- Test for equivalent functionality with Cheb's Necromancy
